custodia/httpd/server.py

```diff
@@ -85,6 +86,42 @@ def server_bind(self):
         self.socket_file = self.socket.getsockname()
 
 
+class ForkingTLSServer(ForkingHTTPServer):
+    def __init__(self, server_address, handler_class, config, sslcontext=None):
```

Comment:
Python's stdlib uses *context* as argument name for a SSLContext object.. python-requests will soon use *sslcontext*. Yet another spelling is just confusing. I'm going for *context* now.